master/pkg/model/environment_config.go

```diff
@@ -176,8 +176,6 @@ func validatePodSpec(podSpec *k8sV1.Pod) []error {
 						"container Ports are not configurable"),
 					check.Equal(len(container.EnvFrom), 0,
```

Comment:
should this check be here still?

Reply:
fixed! Interestingly, this did not stop me from setting `EnvFrom`.

Reply:
this validation would be what run for commands/shells/etc still